src/model/core.py

```diff
@@ -2,25 +2,59 @@
 import copy
 import numpy as np
 
-def erode(im, size):
-    eroded = copy.copy(im)
-    if size > 1:
-        for i in range(len(im)):
-            eroded[i] = morphology.erosion(im[i], morphology.square(size))
+def erode(im, size, round_shape=True):
+    """
+    return an eroded image
+    im: 2d or 3d array
+    size:   int:    generate an element of size (size*2+1)
+    round_shape: str:    if True the elemnt is ball or disk
+    """
```

Comment:
What type of docstring do you use?
I used to write docstring in numpy style or Google style, we should be consistent across all the projet

Reply:
I think we should use the numpy style. Do I update docstrings on the main branch directly?

Reply:
Yes I think its better, it's not really relevant here.